src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -269,6 +273,11 @@ object DemuxFastqs {
       |
       |1. The file extension will be `_R1_001.fastq.gz` for read one, and `_R2_001.fastq.gz` for read two (if paired end).
       |2. The per-sample output prefix will be `<SampleName>_S<SampleOrdinal>_L<LaneNumber>` (without angle brackets).
+      |
+      |To mask bases that are below a specified quality threshold, use the below command, where X is the desired threshold.
+      |Defaults to 0 to keep all bases
+      | --quality-threshold X
```

Comment:
1. Remember that the usage is in markdown, so you can use back-ticks.  I think the is much clearer.
```suggestion
      |Use the `--quality-threshold` option to mask bases that are below a specified quality threshold.
```
2. More importantly, I don't think you need to describe this option to the usage at all.  And if so, it doesn't belong in the Illumina FASTQ File Naming section.